### PR TITLE
Estilos de botones e intro de evento

### PR DESCRIPTION
**Pull request by juanpabloe**
#7 

screen shot 2015-01-20 at 1 52 25 pm

### CODE REVIEW

**Comment by rodowi** [comment]
revisando ...

File: css/styles.css [hunk 1]
```diff
@@ -15,7 +15,7 @@ body {
 }
 
 p {
-	font-family: "Open Sans";
+	font-family: "Open Sans" !important;
```
Comment (rodowi) [inline css/styles.css]: tsss

**Comment by rodowi** [comment]
screen shot 2015-01-20 at 5 37 44 pm

**Comment by rodowi** [comment]
Falta corregir algo de responsiveness

**Comment by juanpabloe** [comment]
so sorry Your Grace

**Comment by rodowi** [comment]
:+1:

**Comment by rodowi** [comment]
stark

**Pull request merged**